docs/manual/architecture/patterns/isolation.md

```diff
@@ -55,32 +50,32 @@ the messages after being encrypted, new keys are generated each time your applic
 To make it easier to follow, here's an ordered list with the approximate steps an IPC message will go through when being
 sent to Tauri Core with the Isolation pattern:
 
-1. Tauri's IPC handler receives message
-2. IPC handler -> Isolation application
-3. `[sandbox]` Isolation application hook runs and potentially modifies the message
-4. `[sandbox]` Message is encrypted with AES-GCM using a runtime-generated key
-5. `[encrypted]` Isolation application -> IPC handler
-6. `[encrypted]` IPC handler -> Tauri Core
+1.  Tauri's IPC handler receives message
+2.  IPC handler -\> Isolation application
+3.  `[sandbox]` Isolation application hook runs and potentially modifies the message
+4.  `[sandbox]` Message is encrypted with AES-GCM using a runtime-generated key
+5.  `[encrypted]` Isolation application -\> IPC handler
+6.  `[encrypted]` IPC handler -\> Tauri Core
 
-_Note: Arrows (->) indicate message passing._
+*Note: Arrows (-\>) indicate message passing.*
 
 ### Performance Implications
 
 Because encryption of the message does occur, this does mean that there are additional overhead costs compared to the
-[Brownfield pattern](./brownfield.md) even if the secure Isolation application doesn't do anything. Outside
+[Brownfield pattern] even if the secure Isolation application doesn't do anything. Outside
 performance-sensitive applications (who likely have a carefully kept small set of dependencies to keep the performance
 adequate), most applications should not notice the runtime costs of encrypting/decrypting the IPC messages as they are
 both relatively small and AES-GCM is relatively fast. If you are unfamiliar with AES-GCM, all that is relative in this
 context is that it's the only authenticated mode algorithm included
-in [SubtleCrypto](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto)
+in [SubtleCrypto]
 and that you probably already use it every day under the hood with TLS.
 
 There is also a cryptographically secure key generated once each time the Tauri application is started. It is not
 generally noticeable if the system already has enough entropy to immediately return enough random numbers, which is
 extremely common for desktop environments. If running in a headless environment to perform
-some [integration testing with WebDriver](../../testing/webdriver/introduction.md)
+some [integration testing with WebDriver]
 then you may want to install some sort of entropy generating service such as `haveged` if your operating system does not
-have one included. <sup>Linux 5.6 (March 2020) now includes entropy generation using speculative execution.</sup>
+have one included. `<sup>`{=html}Linux 5.6 (March 2020) now includes entropy generation using speculative execution.`</sup>`{=html}
```

Comment:
what is this change meant to do? it breaks the superscript rendering. if that was intended just make it it's own line and remove the superscript tags
![image](https://user-images.githubusercontent.com/3301043/154484642-3c298f75-521b-4436-be68-e6d9592324aa.png)

Reply:
Stupid pandoc, fixed!